hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -1303,6 +1304,17 @@ func (r *NodePoolReconciler) getConfig(ctx context.Context,
 		configs = append(configs, *configConfigMap)
 	}
 
+	// Look for operator generated MachineConfigs from the hosted control plane namespace
+	operatorGeneratedConfigs := &corev1.ConfigMapList{}
+	if err := r.List(ctx, operatorGeneratedConfigs, client.MatchingLabels{
+		nodePoolOperatorCreatedConfigLabel: "true",
+		nodePoolAnnotation:                 nodePool.GetName(),
```

Comment:
please don't use an "annotation" name for a label. Also that annotation is meant to have the "ns/name" as value.

Reply:
I fixed this.  I am now using hyperv1.NodePoolLabel which is the exact label I am getting from the Nodes on the NTO side, and copying to the ConfigMap.